### ISSUE TITLE
Need way to validate generated RSpec code

### ISSUE THREAD

**Issue by danascheider** [danascheider/rambo#11]
Although Rambo currently uses Cucumber and RSpec to test that the expected RSpec code is generated, we currently do not have a way to validate that that code is actually valid RSpec that will run without errors. Any suggestions are appreciated, otherwise I will figure this one out at some point.

**Comment by zeisler** [comment]
It could be run through a the ruby parser gem to validate that is valid ruby code. But to lint the `RSpec` DSL without running the code I'm not sure if that could be done.

**Comment by danascheider** [comment]
Yeah, I'm thinking for now that I may do some manual testing - just set up a simple Sinatra API or whatever and test that the RSpecs run. It would also be helpful to explicitly support certain RSpec versions since currently it just generates something RSpec-y
